localstack/services/awslambda/packages.py

```diff
@@ -9,7 +9,7 @@
 
 LAMBDA_RUNTIME_INIT_URL = "https://github.com/localstack/lambda-runtime-init/releases/download/{version}/aws-lambda-rie-{arch}"
 
-LAMBDA_RUNTIME_DEFAULT_VERSION = "v0.1.11-pre"
+LAMBDA_RUNTIME_DEFAULT_VERSION = "v0.1.12-pre"
```

Comment:
This is the key change adopting the new RIE release based with https://github.com/localstack/lambda-runtime-init/pull/13/files